sklearn/decomposition/_fastica.py

```diff
@@ -223,6 +223,16 @@ def my_g(x):
         Initial un-mixing array of dimension (n.comp,n.comp).
         If None (default) then an array of normal r.v.'s is used.
 
+    whiten_solver : str, default='svd'
+        The solver to use for whitening. Can either be 'svd' or 'eigh'.
+
+        * 'svd' is more stable numerically if the problem is degenerate, and
+        often faster when `num_samples<=num_features`.
```

Comment:
Sphinx warning since this is a list and needs to be indented.

```suggestion
        - 'svd' is more stable numerically if the problem is degenerate, and
          often faster when `num_samples<=num_features`.
```